lib/ui/cards/student_info/student_info_card.dart

```diff
@@ -56,28 +58,39 @@ class _StudentInfoCardState extends State<StudentInfoCard> {
     var tokenQueryString =
         "token=" + '${_userDataProvider.authenticationModel.accessToken}';
     var url = _url + "?" + tokenQueryString;
-    return Column(
-      children: <Widget>[
-        Flexible(
-            child: WebView(
-          javascriptMode: JavascriptMode.unrestricted,
-          initialUrl: url,
-          onWebViewCreated: (controller) {
-            _webViewController = controller;
-          },
-          javascriptChannels: <JavascriptChannel>[
-            _printJavascriptChannel(context),
-          ].toSet(),
-        )),
-      ],
-    );
+    print(url);
+    return Container(
+        height: _contentHeight,
+        child: WebView(
+            javascriptMode: JavascriptMode.unrestricted,
+            initialUrl: url,
+            onWebViewCreated: (controller) {
+              _webViewController = controller;
+            },
+            javascriptChannels: <JavascriptChannel>[
+              _printJavascriptChannel(context),
+            ].toSet(),
+            onPageFinished: (some) async {
+              double height = double.parse(await _webViewController
+                  .evaluateJavascript("document.documentElement.offsetHeight"));
+              setState(() {
+                if (height >= cardContentMinHeight &&
+                    height <= cardContentMaxHeight) {
```

Comment:
should _contentHeight be set to cardContentMaxHeight if the new height is larger than max? With this current logic the card height will stay min when the height is extra big.

Reply:
The original feedback from Charles said to set _contentHeight to the Min height. I can do either one.

Reply:
I think that's fine for initialize, but I think the behavior I described should be better for taller content. @c3bryant to confirm that behavior.